vertex_lcfi/util/inc/memorymanager.h

```diff
@@ -72,18 +72,18 @@ namespace vertex_lcfi
 	//Protect the constructor, copy and assignment to prevent usage.		
 	protected:
 		//! Do not use
-		MemoryManager<T>() {}
+		MemoryManager() = delete;
```

Comment:
```suggestion
		MemoryManager() = default;
```

It seems I have been to quick. This is a singleton and it needs the protected constructor, e.g.: https://github.com/iLCSoft/LCFIVertex/actions/runs/7503596352/job/20428683638?pr=9#step:4:331